src/FilamentManager.php

```diff
@@ -193,4 +195,14 @@ public function version()
 
         return InstalledVersions::getPrettyVersion('filament/filament');
     }
+
+    public static function ignoreMigrations()
```

Comment:
```suggestion
    public function ignoreMigrations()
```

Everything is being proxied through the facade, so no need to make this static.